Microsoft.Toolkit.Uwp/Helpers/ObjectStorage/LocalObjectStorageHelper.cs

```diff
@@ -12,12 +12,24 @@ namespace Microsoft.Toolkit.Uwp.Helpers
     public class LocalObjectStorageHelper : BaseObjectStorageHelper
     {
         /// <summary>
-        /// Initializes a new instance of the <see cref="LocalObjectStorageHelper"/> class.
+        /// Initializes a new instance of the <see cref="LocalObjectStorageHelper"/> class,
+        /// which can read and write data using the provided <see cref="IObjectSerializer"/>.
         /// </summary>
-        public LocalObjectStorageHelper()
+        /// <param name="objectSerializer">The serializer to use.</param>
+        public LocalObjectStorageHelper(IObjectSerializer objectSerializer)
+            : base(objectSerializer)
         {
             Settings = ApplicationData.Current.LocalSettings;
             Folder = ApplicationData.Current.LocalFolder;
         }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LocalObjectStorageHelper"/> class,
+        /// which can read and write serialized data using a default Json serializer.
+        /// </summary>
+        public LocalObjectStorageHelper()
+            : this(new JsonObjectSerializer())
```

Comment:
>: this(new JsonObjectSerializer()) [](start = 12, length = 34)

You should be able to remove this and let the compiler pick the default `BaseObjectStorageHelper` constructor